src/js/landing/GoldifyLandingPage.js

```diff
@@ -2,6 +2,10 @@ import React, { Component } from "react";
 import "../../css/GoldifyLandingPage.css";
 
 class GoldifyLandingPage extends Component {
+  /**
+   * Renders the initial div when a user arrives at the site
+   * @returns {?} A div instructing the user on how to get started
```

Comment:
Not sure what to call this return type, looking for suggestions here

Reply:
HTML element? Or perhaps a sub-component?

Reply:
Yeah I think HTML Element would work, i'll update these